MetaData/work/prepareCrabJobs.py

```diff
@@ -109,7 +109,16 @@ def dumpCfg(options):
                     dest="globalTags",
                     action="store",type="string",
                     default="campaigns/globalTagsLookup.json",
-                    help="List of global tags to be used for data and MC. Default: %default")
+                    help="List of global tags to be used for data and MC. Default: %default"),
+        
+        # include additional parameters for cmsRun, such as the parameters from microAODCustomize
+        # the default here is puppi=0 (as an example) see microAODCustomize_cfg.py for more detail
+        make_option("-e","--extraPyCfgParam",
+                    dest="extraPyCfgParam",
+                    action="store",type="string",
+                    default="puppi=0",
```

Comment:
in fact, default should be None, or the if options.extraPyCfgParam would always be true.